app/views/test/saml_test/decode_response.html.erb

```diff
@@ -1,49 +1,45 @@
-<div class="container">
-  <div class="row">
-    <h3>Decoded SAML Response</h3>
+<h3>Decoded SAML Response</h3>
 
-    <p>
-      Visit the account page to demo the SP iniated email change + redirect:
-      <%= link_to 'Account page', account_path, role: 'menuitem' %>
-    </p>
+<p>
+  Visit the account page to demo the SP iniated email change + redirect:
+  <%= link_to 'Account page', account_path, role: 'menuitem' %>
+</p>
 
-    <table border="2" width="80%">
-      <tbody>
-        <tr>
-          <td>IsValid</td>
-          <td>
-            <%= is_valid %>
-          </td>
-        </tr>
-        <tr>
-          <td>Issuer</td>
-          <td>
-            <%= response.settings.idp_entity_id %>
-          </td>
-        </tr>
-        <tr>
-          <td>XML Document</td>
-          <td>
-            <% xml_doc = Base64.encode64(response.document.to_s) %>
-            <%= link_to 'Open in New Window',
-                        "data:text/xml;charset=utf-8;base64,#{xml_doc}",
-                        target: '_blank', rel: 'noopener noreferrer' %>
-          </td>
-        </tr>
-        <tr>
-          <td>Decrypted XML Document</td>
-          <% if response.decrypted_document.present? %>
-            <td>
-              <% xml_doc = Base64.encode64(response.decrypted_document.to_s) %>
-              <%= link_to 'Open in New Window',
-                          "data:text/xml;charset=utf-8;base64,#{xml_doc}",
-                          target: '_blank', rel: 'noopener noreferrer' %>
-            </td>
-          <% else %>
-            <td>Document not encrypted</td>
-          <% end %>
-        </tr>
-      </tbody>
-    </table>
-  </div>
-</div>
+<table border="2" width="80%">
```

Comment:
> should we put the design system table class here usa-table?

Since it's a test page, I'm not overly concerned about it, since it wasn't really a goal of these changes to update the page appearance. This is another one which is carried over from what existed previously, aside from indentation. Also, I have a sense that nobody has looked at this page in ages, so I'd probably sooner remove the route altogether than to spend much time making it look nice 😄